doc/man7/OSSL_PROVIDER-FIPS.pod

```diff
@@ -80,6 +82,10 @@ The OpenSSL FIPS provider supports these operations and algorithms:
 
 =item AES, see L<EVP_CIPHER-AES(7)>
 
+=item 3DES, see L<EVP_CIPHER-DES(7)>
+
+This is an unapproved algorithm
```

Comment:
Full stop at the end.

Should we also include a note: _this will be removed in the next major release_?
We do not have a policy regarding FIPS algorithm removals -- typically the process is insanely slow but not quite as slow as our deprecation policy.  We've also no way to _deprecate_ things in the FIPS provider.  The best we've got is a config flag to twiddle things.

Reply:
Full stop added.

I would defer adding that note for now. Probably that needs discussion, so I'd put any such note in a separate PR.